### ISSUE TITLE
Feedback on new traffic signs controls

### ISSUE THREAD

**Issue by de-vries** [mapillary/mapillary_issues#774]
The traffic signs controls just disappeared on both the game and the map view

**Comment by knikel** [comment]
@de-vries Have you tried clearing cache and shift-refreshed?

**Comment by de-vries** [comment]
I tried restarting ff and logging in/out, didn't work

I now just tried to refresh without chase, they came back, but oddly placed (a bit to low).
then I disabled my userContent.css and restarted ff and now I can't get the controls back.

**Comment by knikel** [comment]
@de-vries yes, the controls could be a bit off placed, as I'm in the process of gradually updating the whole traffic game view. So expect some visual changes soon ;)

**Comment by de-vries** [comment]
> So expect some visual changes soon ;) 

Can't wait :)

I got the controls back again, but indeed a bit off placed. I'll be patient then

btw: now the in photo change sign button doesn't work, If you want further testing just ask, I'm back in a few hours.

**Comment by knikel** [comment]
@de-vries I've deployed some big brush layout changes, how is it working for you now?

**Comment by de-vries** [comment]
Testing now:

nice work, looks much better now. 
I guess you can close issue https://github.com/mapillary/mapillary_issues/issues/356 now :)
Here are some minor issues:
- the in photo change sign buttons still don't work.
- When a new photo is loading, the signs and buttons from the previous image are shown.
- The height of the button bar collapses when going to a new photo. (as the buttons temporarily disappear)

**Comment by knikel** [comment]
Thanks for the feedback @de-vries, investigating your points now.

**Comment by de-vries** [comment]
I have some more :)
Firstly I made some points few weeks ago: https://github.com/mapillary/mapillary_issues/issues/633
Those points are not directly related to your work of the last 24 hours, but close. Especially the last point from that issue should be only 5 seconds of coding:
> The title of the 'Select Traffic Signs' pop-up should be: 'Select Traffic Sign' (singular)

**Comment by de-vries** [comment]
Last critique:

Since #356 is solved now, there is no need to zoom out anymore after an edit. I think that this raises the priority of issue #703.

Also with issue #356 gone, there is no indicator anymore to see when you are at "zoom 100%". I think that we need a button to go to"zoom 100%" that changes color when at 100%, so it doubles as an indicator.
See also issue #517.

This would be a good icon for that: ![selectie_046](https://cloud.githubusercontent.com/assets/1449609/6483158/0aac39da-c26e-11e4-9828-a056e471b41d.png). Sadly I couldn't find anything useful in Font Awesome. (are you using any other icon fonts?)

**Comment by de-vries** [comment]
When hovering over the marker in the photo the is some feedback at the corresponding sign in the button bar and vice-versa.

When hovering over newly made signs in the button bar this doesn't work.

**Comment by knikel** [comment]
Awesome @de-vries, thanks a lot for the feedback! So in the next release:

> the in photo change sign buttons still don't work.

:zap: fixed

>  The height of the button bar collapses when going to a new photo. (as the buttons temporarily disappear)

:zap: fixed

> The title of the 'Select Traffic Signs' pop-up should be: 'Select Traffic Sign' (singular)

:zap: fixed

Still need to research the previous photo rectangles flashing on a new photo.

Regarding #703, it seems that some people prefer to keep the zoom level 'locked', others don't. I'll have a look what might be a viable option in that regard. Your button suggestion is a good one. Speaking of the icon font, yes, the latest version of FontAwesome is the only font we use. I would prefer to stay within the FontAwesome range of icons, maybe [fa-compress](http://fontawesome.io/icon/compress) with `Reset zoom level` string could do; it would appear somewhere around the image only when user is at non-100% zoom level. I'll research that further. Thanks for the suggestion though!

> When hovering over the marker in the photo the is some feedback at the corresponding sign in the button bar and vice-versa.
> When hovering over newly made signs in the button bar this doesn't work.

Thanks, addressing this in the next release.

**Comment by de-vries** [comment]
I like the new hover state of the overlay rectancle, but it's a bit jumpy cause the border in hoverstate gets added to the size. to fix it, give the non hoverstate also a border:

`.mapillaryimagevieworoverlay__overlay__rect {
    border: 1px solid rgba(255, 0, 0, 0.7);
}`

**Comment by knikel** [comment]
@de-vries yes, good point, I missed that one, thanks!

**Comment by de-vries** [comment]
> > The height of the button bar collapses when going to a new photo. (as the buttons temporarily disappear)

> :zap: fixed

Well mostly fixed, the signs are a bit higher than the buttons and if you remove the last sign the button bar collapses a little.
this should fix it:

`.or-controls__tool-container {
    margin: 7px 0px;
}`

In the map view 6px is already enough.

**Comment by de-vries** [comment]
@knikel When the game was introduced I made a comment with a list of remarks, a lot of them got answered or solved, but I believe there are still some valid points in it. Today I've updated it and since you seem to be the one responsible for the traffic sing controls I want to bring it under your attention. I'm talk in about [this](https://github.com/mapillary/mapillary_issues/issues/568#issuecomment-74046011) comment.
